lib/rucio/core/trace.py

```diff
@@ -38,7 +38,7 @@
 
 from rucio.common.config import config_get, config_get_int
 from rucio.common.exception import InvalidObject
-from rucio.common.schema.generic import SCOPE, RSE, UUID, TIME_ENTRY, IP, CLIENT_STATE
+from rucio.common.schema.generic import SCOPE, RSE, UUID, TIME_ENTRY, IPv4orIPv6, CLIENT_STATE
```

Comment:
We should address this at a later point, but at least SCOPE, RSE should not come from the generic schema, it needs to come from the VO specific ones.

Reply:
#5105 should take care of the required refactoring.